vllm/entrypoints/openai/api_server.py

```diff
@@ -474,7 +511,23 @@ async def show_version():
 
 
 @router.post("/v1/chat/completions",
-             dependencies=[Depends(validate_json_request)])
+             dependencies=[Depends(validate_json_request)],
+             responses={
+                 HTTPStatus.OK.value: {
+                     "content": {
+                         "text/event-stream": {}
+                     }
```

Comment:
Needed to test expecting `Content-Type: application/json` all the time. See https://fastapi.tiangolo.com/advanced/additional-responses/#additional-media-types-for-the-main-response